icicle/appUtils/lde/lde.cuh

```diff
@@ -6,61 +6,57 @@
 
 /**
  * @namespace lde
- * LDE (stands for low degree extension) contains [NTT](@ref ntt)-based methods for translating between coefficient and
- * evaluation domains of polynomials. It also contains methods for element-wise manipulation of vectors, which is useful
- * for working with polynomials in evaluation domain.
+ * LDE (stands for low degree extension) contains [NTT](@ref ntt)-based methods for translating between coefficient and evaluation domains of polynomials.
```

Comment:
isn't LDE a part of NTT and not vice versa?

Reply:
Partially answered in the previous comment: I imagine future LDE as something like arkworks' [`Evaluation`](https://github.com/arkworks-rs/algebra/blob/master/poly/src/evaluations/univariate/mod.rs) or a mix of `Evaluation` and [`DensePolynomial`](https://github.com/arkworks-rs/algebra/blob/master/poly/src/polynomial/univariate/dense.rs). So, it's a higher-level wrapper over NTT. For example, to evaluate a polynomial of degree `N` on a domain of size `2N` you need 2 size `N` NTTs, or to multiply two polynomials you need some NTTs for translating between evaluation and coefficient forms